### PR TITLE
Added feature tests for Sign-up

### PR DESCRIPTION
**Pull request by MrJadaml**
Add specs for UI behavior #9

### CODE REVIEW

File: spec/spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```
Comment (awong-dev) [inline spec/spec/spec_helper.rb]: This file seems odd.  Why is it spec/spec? Can't we just use the spec/spec_helper.rb that's already there?

File: spec/features/devise/registration_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+feature "Registration" do
```
Comment (awong-dev) [inline spec/features/devise/registration_spec.rb]: Can we continue the commenting pattern as here:
  https://github.com/awong-dev/SCIInterim/blob/master/spec/features/patients/registry_spec.rb#L1

Curious to see how onerous it is.  The idea of writing these comments is it forces the spec to stay focused on particular assertion.

If they end up just repeating what we write in the scenario text too much we can drop it...but for now would like to push the trend a little while longer.

Example of such assertions being used to force focus on tests in my previous project:
  https://github.com/adobe/chromium/blob/master/base/bind_unittest.cc#L322

It's C++ and the test is not UI, but the idea is still there.

File: spec/spec/spec_helper.rb (hunk 1 above)
Comment (MrJadaml) [inline spec/spec/spec_helper.rb]: probably popped that extra one on there when I was getting rspec up and running. Looks like you cleared it now.

File: spec/features/devise/registration_spec.rb (hunk 2 above)
Comment (MrJadaml) [inline spec/features/devise/registration_spec.rb]: Yup. I will move forward with specs using that commenting pattern.

**Comment by awong-dev** [comment]
awesomepossum.

**Comment by awong-dev** [comment]
Just making sure we aren't deadlocked...I'm waiting on responses to comments before reviewing again. Ping me when it's ready for another look!

File: spec/spec/spec_helper.rb (hunk 1 above)
Comment (awong-dev) [inline spec/spec/spec_helper.rb]: ...seems to still be in the pull request. Let me just clone down and get all the git stuff straight. I'll probably fork into a new PR.

**Comment by awong-dev** [comment]
gha...I completely screwed up your login why typing @MrJadaml. Sorry! :(

**Pull request closed without merging**